### ISSUE TITLE
[Help] I can't set up the find command correctly

### ISSUE THREAD

**Issue by schemar** [andreyorst/fzf.kak#90]
I have the following in my `kakrc`, but I am doing it wrong. Using the file finder from `fzf` mode lists `.git/` files. What am I doing wrong?

Thank you for your help.

My `kakrc` block, which includes `set-option global fzf_file_command 'fd --type f --hidden --exclude ".git/"'`:

```
plug "https://github.com/andreyorst/fzf.kak" tag "v1.0.3" config %{
    map global user f ': fzf-mode<ret>' -docstring 'fuzzy find'
} defer fzf %{
    map global normal <c-p> ': fzf-mode<ret>'
    set-option global fzf_use_main_selection false
} defer fzf-file %{
    set-option global fzf_file_command 'fd --type f --hidden --exclude ".git/"'
    set-option global fzf_grep_command rg
    set-option global fzf_highlight_command 'bat --style=numbers --color=always {}'
} 
```

Running `fd --type f --hidden --exclude ".git/"` from the command line works as expected.

**Comment by schemar** [comment]
I tried again with latest `master` and it works now. It doesn't work with [release 1.0.3](https://github.com/andreyorst/fzf.kak/releases/tag/v1.0.3), though.

**Comment by andreyorst** [comment]
> It doesn't work with release 1.0.3, though.

I should release 1.0.4 then

**Comment by fraunos** [comment]
I've got the same problem and I see 1.0.4 isn't released yet. Can we reopen this?

**Comment by andreyorst** [comment]
@fraunos sorry for the inconvenience, I've released `v1.1.0`. This change is too big for minor release

**Comment by fraunos** [comment]
@andreyorst You're awesome, thanks!
